keylime/mba/mba.py

```diff
@@ -21,46 +46,98 @@
 MBAgg = Optional[Dict[str, List[str]]]
 
 # ###########
-# import/load policy engine
+# list of all imported modules for MBA.
 # ###########
 
+_mba_imports = []
 
-def load_policy_engine() -> None:
+# ###########
+# import/load measured boot attestation imports
+# ###########
+
+
+def load_imports() -> None:
     """
-    MBA API front-end for importing the policy engine.
-    No inputs, but implementation *will* read from keylime configuration files.
+    MBA API front-end for importing any modules needed by measured boot attestation.
+    No inputs.
     No outputs.
-    Side effects: policy engine is now loaded.
-    Exceptions: If the policy engine cannot be loaded.
+    Side effects: Reads from keylime configuration files.
+      After execution all imports required by MBA are loaded and initialized.
+    Exceptions: If any policy engine in the list cannot be loaded.
+    """
+    try:
+        imports = config.getlist("verifier", "measured_boot_imports")
+        # these are the defaults
+        imports.append("keylime.mba.elchecking.elchecker")
+        imports.append("keylime.mba.elchecking.example")
+        imports.append("keylime.mba.elparsing.tpm2_tools_elparser")
```

Comment:
Those that are always in the list, should unconditionally imported with "import". Why to do that dynamically?

Reply:
I want these modules in the list because `_find_implementation` will scan that list to find functions to call. There might be nicer ways to implement this, but for now having an explicit list works.

Reply:
oh bummer :+1: